src/app/containers/SubHeading/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SubHeadingContainer with data should display the provided sub-heading 1`] = `
+<h2
+  className=""
+>
+  The amazing sub-heading!?
+</h2>
+`;
+
+exports[`SubHeadingContainer with data should display the subheading containing various symbols 1`] = `
+<h2
+  className=""
+>
+  !@#$%^&*()'"?/[]{}
```

Comment:
@radiocontrolled I am assuming that it is because this test is testing a specific feature which is specific to our particular use case of the component.